src/main/java/com/iabtcf/decoder/ConsentParser.java

```diff
@@ -0,0 +1,311 @@
+    IntIterable getVendorLegitimateInterest();
+
+    /**
+     * The restrictions of a vendor's data processing by a publisher within the context of the users
+     * trafficking their digital property.
+     * 
+     * @since 2.0
+     * @return the list of publisher restrictions.
+     */
+    List<PublisherRestriction> getPublisherRestrictions();
+
+    /**
+     * Part of the OOB segments expressing that a Vendor is using legal bases outside of the TCF to
+     * process personal data.
+     * 
+     * @since 2.0
+     * @return A list of Vendors that the publisher allows to use out-of-band legal bases.
+     */
+    IntIterable getAllowedVendors();
+
+    /**
+     * Part of the OOB segments expressing that a Vendor is using legal bases outside of the TCF to
+     * process personal data.
+     * 
+     * @since 2.0
+     * @return A list of Vendors that disclosed to the user.
+     */
+    IntIterable getDisclosedVendors();
+
+    /**
+     * Part of the Publisher Transparency and Consent segment of a TC String that publishers may use
+     * to establish transparency with and receive consent from users for their own legal bases to
+     * process personal data or to share with vendors if they so choose.
+     * 
+     * The user's consent value for each Purpose established on the legal basis of consent, for the
+     * publisher
+     * 
+     * The Purposes are numerically identified and published in the Global Vendor List.
+     * 
+     * @since 2.0
+     * @return the consent value for each Purpose
+     */
+    IntIterable getPubPurposesConsent();
+
+    /**
+     * Part of the Publisher Transparency and Consent segment of a TC String that publishers may use
+     * to establish transparency with and receive consent from users for their own legal bases to
+     * process personal data or to share with vendors if they so choose.
+     * 
+     * The Purpose’s transparency requirements are met for each Purpose established on the legal
+     * basis of legitimate interest and the user has not exercised their "Right to Object" to that
+     * Purpose.
+     * 
+     * By default or if the user has exercised their "Right to Object" to a Purpose, the
+     * corresponding identifier for that Purpose is set to false.
+     * 
+     * @since 2.0
+     * @return The consent value for each Purpose where legitimate interest was established.
+     */
+    IntIterable getPubPurposesLITransparency();
+
+    /**
+     * Part of the Publisher Transparency and Consent segment of a TC String that publishers may use
+     * to establish transparency with and receive consent from users for their own legal bases to
+     * process personal data or to share with vendors if they so choose.
+     * 
+     * Custom purposes will be defined by the publisher and displayed to a user in a CMP user
+     * interface.
+     * 
+     * @since 2.0
+     * @return The established custom purpose consent values
+     */
+    IntIterable getCustomPurposesConsent();
```

Comment:
include the Pub word in this and the method below it.

Reply:
We already have names that start with "pub" which would add confusion. Calling the method as defined in the spec is desirable.

Reply:
Seeing as it comes from the Pub segment, I think it would actually decrease confusion. Since we have other methods with the Pub word in it, based on the method name now, this would make one think this is not related to publisher specific consents until you look at the java doc.

Reply:
then getPubPurposesLITransparency would be getPubPubPurposesLITransparency.